MsbuildDependencyVisualizer.UI/MsbuildDependencyVisualizer.UI.csproj

```diff
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>MsbuildDependencyVisualizer.UI</RootNamespace>
     <AssemblyName>MsbuildDependencyVisualizer.UI</AssemblyName>
-    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```

Comment:
Again, please use the latest .net framework. Looks like you have the older version.